examples/seismic/tutorials/11_viscoacoustic.ipynb

```diff
@@ -0,0 +1,926 @@
+    "# Time step in ms and time range:\n",
+    "t0, tn = 0., 2000.\n",
+    "dt = model.critical_dt\n",
+    "time_range = TimeAxis(start=t0, stop=tn, step=dt)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from examples.seismic import Receiver\n",
+    "\n",
+    "def src_rec(p, model):\n",
+    "    src = RickerSource(name='src', grid=model.grid, f0=f0, time_range=time_range)\n",
+    "    src.coordinates.data[0, :] = np.array(model.domain_size) * .5\n",
+    "    src.coordinates.data[0, -1] = 8.  \n",
+    "\n",
+    "    # Create symbol for receivers\n",
+    "    rec = Receiver(name='rec', grid=model.grid, npoint=shape[0], time_range=time_range)\n",
+    "\n",
+    "    # Prescribe even spacing for receivers along the x-axis\n",
+    "    rec.coordinates.data[:, 0] = np.linspace(0, model.domain_size[0], num=shape[0])\n",
+    "    rec.coordinates.data[:, 1] = 8.  \n",
+    "\n",
+    "    src_term = src.inject(field=p.forward, expr=(s*src))\n",
+    "    rec_term = rec.interpolate(expr=p)\n",
+    "    \n",
+    "    return src_term + rec_term, src, rec"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Auxiliary functions for plotting data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_receiver(rec):\n",
+    "    rec_plot = rec.resample(num=1001)\n",
+    "    scale_for_plot = np.diag(np.linspace(1.0, 2.5, 1001)**2.0)\n",
+    "    # Pressure (txx + tzz) data at sea surface\n",
+    "    extent = [rec_plot.coordinates.data[0, 0], rec_plot.coordinates.data[-1, 0], 1e-3*tn, t0]\n",
+    "    aspect = rec_plot.coordinates.data[-1, 0]/(1e-3*tn)/.5\n",
+    "\n",
+    "    plt.figure(figsize=(10, 10))\n",
+    "    plt.imshow(np.dot(scale_for_plot, rec_plot.data), vmin=-.01, vmax=.01, cmap=\"seismic\",\n",
+    "           interpolation='lanczos', extent=extent, aspect=aspect)\n",
+    "    plt.ylabel(\"Time (s)\", fontsize=20)\n",
+    "    plt.xlabel(\"Receiver position (m)\", fontsize=20)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_v_and_p(model, v, p):\n",
+    "    \n",
+    "    slices = [slice(model.nbl, -model.nbl), slice(model.nbl, -model.nbl)]\n",
+    "    scale = .5*1e-3\n",
+    "\n",
+    "    plt_options_model = {'extent': [model.origin[0] , model.origin[0] + model.domain_size[0],\n",
+    "                                    model.origin[1] + model.domain_size[1], model.origin[1]]}\n",
+    "\n",
+    "    fig, ax = plt.subplots(nrows=1, ncols=3, figsize=(15, 7))\n",
+    "\n",
+    "    ax[0].imshow(np.transpose(v[0].data[0][slices]), vmin=-scale, vmax=scale, cmap=\"RdGy\", **plt_options_model)\n",
+    "    ax[0].imshow(np.transpose(model.vp.data), vmin=1.5, vmax=3.5, cmap=\"jet\", alpha=.5, **plt_options_model)\n",
```

Comment:
`model.vp.data[slices]` and same below, you can see on the images that the reflection in the wavefield doesn't align with the refelctor in the velocity.

Reply:
Done.